pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computemanagedsslcertificate/managedsslcertificate/create.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 Google LLC
+# Copyright 2024 Google LLC
```

Comment:
No need the parent folder `managedsslcertificate` because there is only one test case: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.NewResourceFromTerraform.md#add-necessary-yaml-files-to-testdata

Reply:
This was mostly copied over from v1alpha1; I can remove the extra dir.